### PR TITLE
added new speaker config

### PR DESCRIPTION
**Pull request by lemmyg**
Added a new entry for new experimental config for macbook pro 16.1 speakers

### CODE REVIEW

File: docs/guides/audio-config.md
Comment (sharpenedblade) [inline docs/guides/audio-config.md:56]: I think it would be better to move this to a sub-heading under `## Speakers`. Something like `### MacBookPro16,1`.

File: docs/guides/audio-config.md
Comment (sharpenedblade) [inline docs/guides/audio-config.md:61]: Just a link to the repo doesn't explain the installation process to users, we should tell them to `git clone` it, then run the install script. It would also be simpler to tell them to run `cp` manually instead of the install script, but that might make it more confusing for new users, and it is not documented in the README.

**Review by sharpenedblade — CHANGES_REQUESTED** [review]
Users expect a download link and step-by-step instructions, so a link to the repo will be confusing.

File: docs/guides/audio-config.md
Comment (AdityaGarg8) [inline docs/guides/audio-config.md:44]: All of apple's fancy tuning of the speakers is done in macOS, similar configuration is currently available only for MacBook Pro 16 inch 2019 only.

This sounds better to me.

File: docs/guides/audio-config.md
Comment (AdityaGarg8) [inline docs/guides/audio-config.md:58]: 16,4 and 16,1 share the same config. So probably add that as well.

File: docs/guides/audio-config.md
Comment (AdityaGarg8) [reply docs/guides/audio-config.md:61]: The readme looks like to me, except that it's missing the dependencies that are required (atleast for Ubuntu).

**Review by AdityaGarg8 — CHANGES_REQUESTED** [review]

File: docs/guides/audio-config.md
Comment (sharpenedblade) [reply docs/guides/audio-config.md:61]: > The readme looks like to me, except that it's missing the dependencies that are required (atleast for Ubuntu).

I think we should add the install steps directly on the wiki, so it is simpler to understand. A user new to this config can easily find the steps, rather than looking through another link.

File: docs/guides/audio-config.md
Comment (Redecorating) [reply docs/guides/audio-config.md:58]: It might be worth saying "DSP (<what DSP stands for>)", as some people (including me!) don't know what that means.

File: docs/guides/audio-config.md
Comment (lemmyg) [reply docs/guides/audio-config.md:61]: Hello,
I added the missing ubuntu deps
https://github.com/lemmyg/asahi-audio/tree/macbookT2_16_1

I believe all the required instructions are in the Readme. I didn't want to duplicate the information.

File: docs/guides/audio-config.md
Comment (lemmyg) [reply docs/guides/audio-config.md]: > 16,4 and 16,1 share the same config. So probably add that as well.

first It should be checked if the use same speakers model

File: docs/guides/audio-config.md
Comment (lemmyg) [reply docs/guides/audio-config.md:44]: ok

File: docs/guides/audio-config.md
Comment (sharpenedblade) [reply docs/guides/audio-config.md]: > Hello, I added the missing ubuntu deps https://github.com/lemmyg/asahi-audio/tree/macbookT2_16_1
> 
> I believe all the required instructions are in the Readme. I didn't want to duplicate the information.

Can you close this review

File: docs/guides/audio-config.md
Comment (sharpenedblade) [reply docs/guides/audio-config.md:44]: I think this is clearer.

All of apple's fancy tuning of the speakers is done in macOS, but a similar configuration is currently available for only the MacBook Pro 16 inch 2019.

File: docs/guides/audio-config.md
Comment (sharpenedblade) [reply docs/guides/audio-config.md:56]: Can you close this

File: docs/guides/audio-config.md
Comment (Redecorating) [inline docs/guides/audio-config.md:62]: maybe note that it does/doesn't work on 16,4 or that it's untested if the same config works on 16,4?

**Review by Redecorating — APPROVED** [review]
lgtm, in the future i wonder if we can have a service that symlinks the config to /run/... at boot before pipewire starts (this way it can check the model every time). Even better would be to get pipewire to know how to pick config based off of the model

File: docs/guides/audio-config.md
Comment (AdityaGarg8) [reply docs/guides/audio-config.md:62]: I'd prefer MacBook Pro 16 inch, 2019 here

File: docs/guides/audio-config.md
Comment (AdityaGarg8) [inline docs/guides/audio-config.md:63]: macOS

File: docs/guides/audio-config.md
Comment (AdityaGarg8) [reply docs/guides/audio-config.md:44]: This also looks good

**Review by AdityaGarg8 — CHANGES_REQUESTED** [review]

File: docs/guides/audio-config.md
Comment (lemmyg) [reply docs/guides/audio-config.md:62]: I prefer to leave it to the exact model
MacBook Pro 16,1 2019

Comment (lemmyg) [reply docs/guides/audio-config.md:62]: About untested models. I'm already saying any other model is not supported and this configuration should not be used for now.

Comment (AdityaGarg8) [reply docs/guides/audio-config.md:62]: But they are same models.

Comment (lemmyg) [reply docs/guides/audio-config.md:62]: what's the different betwen 16,1 and 16,4?

Comment (AdityaGarg8) [reply docs/guides/audio-config.md:62]: The AMD graphics card

Comment (AdityaGarg8) [reply docs/guides/audio-config.md:62]: https://support.apple.com/en-in/HT201300

File: docs/guides/audio-config.md
Comment (solarkennedy) [reply docs/guides/audio-config.md:44]: I'm also glad to see this line replaced!
I have a 16,1 and will try this out eventually, but to be honest I was taken a little aback with the complexity of what it takes to make this work.

But maybe it will inspire others with different models to at least investigate and contribute.

Comment (lemmyg) [reply docs/guides/audio-config.md:44]: this is just a initial step to notify users about the new settings. 
there will be more revision to make the installation more simple

File: docs/guides/audio-config.md
Comment (lemmyg) [reply docs/guides/audio-config.md]: done

File: docs/guides/audio-config.md
Comment (AdityaGarg8) [inline docs/guides/audio-config.md:62]: I believe users will be more comfortable with MacBook Pro 16 inch, 2019 rather than 16,1/16,4

File: docs/guides/audio-config.md
Comment (AdityaGarg8) [inline docs/guides/audio-config.md:60]: There should be a space after DSP

File: docs/guides/audio-config.md
Comment (AdityaGarg8) [inline docs/guides/audio-config.md:61]: each model *needs specific settings*

**Review by AdityaGarg8 — CHANGES_REQUESTED** [review]

**Comment by AdityaGarg8** [comment]
You also have added speakers and MacBook Pro 16 inch 2019 section under to microphone fixes heading. You'll get it when you see the preview of the file. Also, maybe we could make a heading named DSP and add MBP 16" over there.

**Comment by lemmyg** [comment]
> You also have added speakers and MacBook Pro 16 inch 2019 section under to microphone fixes heading. You'll get it when you see the preview of the file. Also, maybe we could make a heading named DSP and add MBP 16" over there.

done

**Review by AdityaGarg8 — APPROVED** [review]

**Pull request merged**
